src/definitions/behaviors/form.js

```diff
@@ -642,6 +642,7 @@ $.fn.form = function(parameters) {
             if((t=$field.filter('[data-' + metadata.validate + '="'+ identifier +'"]')).length > 0 ) {
               return t;
             }
+            module.debug('Field identifier not found.', identifier);
```

Comment:
this error seems to me much more severe

What is the purpose of returning an unattached `$('<input/>')`?

`module.debug` is silent when debug is not enabled - would a change to `module.error` be ok?

the console error would be like:

![image](https://user-images.githubusercontent.com/2228672/194760620-a706ff91-f4f6-492b-910b-0868a74d0a63.png)

and guide the developer to the problem much faster

Reply:
I changed it to an error by #2489 
I believe the purpose of returning the empty input was to not break all the following code, so we leave that as it is for now.